docs/README.md

```diff
@@ -4,34 +4,38 @@ legacyUrl: https://docs.ipfs.io/
+<p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>
 
-IPFS docs are organized into the following main categories:
+## Host a static website on IPFS
 
-## Install IPFS
+Protect your site from a single point of failure by importing it to the IPFS network, and benefit from decentralized infrastructure. If you're new to IPFS, this is an excellent place to start learning how to build on the decentralized web. [Start the tutorial →](/how-to/host-single-page-site/)
 
-[Here's your guide](/install/) if you just want to install IPFS and get started right away!
+## Community projects
 
-## Concepts
+Are you looking for inspiration? You can find a huge collection of awesome community projects over at Awesome IPFS. [Go there now →](https://awesome.ipfs.io/)
 
-[Check out these key IPFS concepts](/concepts/) if you're new to IPFS or the distributed web in general. Because IPFS is a system that hopes to change how we use the Internet, it comes with many new concepts! This section aims to get you up to speed quickly.
+![The Awesome IPFS homepage.](./images/awesome-ipfs.png)
 
-## How-tos
+## Useful tools
 
-[Our how-tos and tutorials](/how-to/) offer more in-depth advice and information on IPFS specifics from customizing your install to managing files, working with peers, building apps, and more.
+Build your apps and services quicker with some help from these tools built for IPFS.
 
-Also, check out [ProtoSchool](https://www.proto.school) for interactive tutorials on decentralized web concepts, protocols, and tools that you can complete right in your browser!
+| [Browser Companion](https://github.com/ipfs-shipyard/ipfs-companion)   | [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop)| [IPLD Explorer](https://explore.ipld.io/) |
+| --- | --- | --- |
+| ![The IFPS browser companion in Firefox.](./images/ipfs-companion.png) | ![The IPFS desktop app running on MacOS with the status tab open.](./images/ipfs-desktop.png) | ![The IPLD Explorer homepage.](./images/ipld-explorer.png) |
 
-## API & CLI reference
+## Decentralized web
 
-[Find developer references](/reference/) to help you get things done — whether you're using IPFS via the command line, interacting with IPFS nodes using HTTP, or creating your own new applications on top of IPFS using Go or JavaScript.
+Want to know more about the decentralized web (Dweb) and how it's changing the internet? These resources will give you a clearer understanding of the Web movement, and how it's improving the world:
 
-## Community
+- [What is IPFS](/concepts/what-is-ipfs/)
+- [Peer-to-peer sharing](/concepts/dht/)
+- [Content-addressed data](/concepts/content-addressing/)
```

Comment:
```suggestion
- [Content-addressed data](/concepts/content-addressing/)

[See more guides in the Concepts section →](/concepts/)
```
Added an arrow-link to the whole section for consistency with other parts of this page